agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -168,6 +169,12 @@ public static class Sampling {
     @Deprecated @Nullable public Double limitPerSecond;
 
     public List<SamplingOverride> overrides = new ArrayList<>();
+
+    private void validate() {
+      for (SamplingOverride samplingOverride : overrides) {
+        samplingOverride.validate();
+      }
+    }
```

Comment:
validation of the GA'd config was missing

Reply:
good catch